test.html

```diff
@@ -164,6 +281,65 @@ <h2>Test drive</h2>
 		});
 }
 
+/**
+ * Loads a plugin including it css file
+ *
+ * @param  {string} plugin the plugin to load
+ * @type {Promise} the promise which resolves as soon as plugin is loaded
+ */
+function loadPlugin (plugin)
+{
+	if (!components.plugins[plugin]){
+		return;
+	}
+
+	var pluginPath = components.plugins.meta.path;
+
+	return new Promise(function (resolve) {
+		pluginPath = pluginPath.replace(/{id}/g, plugin);
+
+		$u.script(pluginPath + '.js', function() {
+			this.setAttribute('data-type', 'plugin');
+
+			var style = document.createElement('link');
+			style.setAttribute('rel', 'stylesheet');
+			style.setAttribute('href', pluginPath + '.css')
+			style.setAttribute('data-type', 'plugin');
+			document.body.appendChild(style);
+
+			resolve();
+		});
+	});
+}
+
+/**
+ * Reload all selected plugins and prism core
+ * You can hit me for such solution, but there are no other way :)
+ */
+function reloadPlugins ()
+{
+	var selectedPlugins = $$('input[type="checkbox"]:checked', document.forms[0]);
+	selectedPlugins = selectedPlugins.map(function(plugin) { return plugin.value });
+
+	$('#prism').remove();
+	$$('script[data-type="plugin"],link[data-type="plugin"]').forEach(function (script) {
+		script.remove();
```

Comment:
@Golmote @LeaVerou Removing this script tag doesn't undo what the script has done. I think we'd need to implement #1381 before we can do this. Any other ideas?

Reply:
We could use an iframe whose `srcset` is set dynamically for the demo :)

PS: Awwww, I wish we could remove `<script>` elements and have their effects undone, that would be so nicely reactive! :)

Reply:
Haha, yes, indeed. It was done (probably? 2 years has passed..) for more clean html structure, I guess.. But later I'm realoading Prism core js file - so it will be run again and we will receive clean object. 

Doing everything in iframes is better and nice idea also